src/main/java/net/masterthought/cucumber/json/DocString.java

```diff
@@ -17,4 +17,8 @@
     public String getValue() {
         return value;
     }
+
+    public String getValueWithBreakingSpace() {
+        return value!=null?value.replaceAll("(\r\n|\r|\n|\n\r)", "<br/>"):value;
```

Comment:
If you want to display it as html code then convert it before passing to the report. Report displays data 'as it is' and no conversion is applied

this is because if someday I will decide to display this in PRE block then such conversion won't be compatible

even worse - if someone adds html code into this doc_string then the expected outcome would be different then generated